src/map/lua/luautils.cpp

```diff
@@ -675,13 +692,16 @@ namespace luautils
 
     /************************************************************************
     *                                                                       *
-    *   Return Vanadiel Day element                                         *
+    *   Return Vanadiel Day's element                                       *
+    *   Note: THIS IS NOT THE SAME AS THE DAY OF THE WEEK                   *
+    *   Days of week: Fire Earth Water Wind Ice Lightning Light Dark        *
+    *   Elements: Fire Ice Wind Earth Thunder Water Light Dark              *
     *                                                                       *
     ************************************************************************/
 
     int32 VanadielDayElement(lua_State* L)
     {
-        lua_pushinteger(L, CVanaTime::getInstance()->getWeekday());
+        lua_pushinteger(L, battleutils::GetDayElement());
```

Comment:
I went back and forth on this. I originally considered having elements be part of the time object, but that didn't feel entirely right:
- Elements are really only referenced in the context of combat (spell elements)
- We currently have GetWeather in `battleutils`, I wanted GetDayElement to be in there with it, and I was hesitant to create yet another header/util for "common world" functions (too much rocking the boat for one PR)